packages/plugins/plugin-space/src/components/RecordArticle.tsx

```
@@ -28,23 +28,25 @@ export const RecordArticle = ({ role, subject }: SurfaceComponentProps) => {
-            />
+      <ScrollArea.Root orientation='vertical'>
+        <ScrollArea.Viewport classNames={mx('p-4 gap-4')}>
+          <div role='none' className={mx('flex is-full card-max-width')}>
+            <Surface.Surface role='section' data={data} limit={1} />
           </div>
-        )}
-      </ScrollArea>
+
+          {related.length > 0 && (
+            <div role='none' className={mx('flex flex-col gap-1', singleColumn ? 'card-max-width' : 'is-full')}>
+              <label className='mbs-2 text-sm text-description'>{t('related objects label')}</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)